bindata/network/multus/multus.yaml

```diff
@@ -135,14 +135,66 @@ data:
         "multusAutoconfigDir": "/host/run/multus/cni/net.d",
         "namespaceIsolation": true,
         "globalNamespaces": "default,openshift-multus,openshift-sriov-network-operator",
+{{- if not .IsNetworkTypeLiveMigration}}
```

Comment:
Is `daemon-config.json` used during migration? If not I don't see why this is needed